cobalt/persistent_storage/persistent_settings.cc

```diff
@@ -113,47 +113,49 @@ bool PersistentSettings::GetPersistentSettingAsBool(const std::string& key,
                                                     bool default_setting) {
   base::AutoLock auto_lock(pref_store_lock_);
   auto persistent_settings = pref_store_->GetValues();
-  const base::Value* result = persistent_settings->FindKey(key);
-  if (result && result->is_bool()) return result->GetBool();
+  absl::optional<bool> result = persistent_settings.FindBool(key);
+  if (result) return *result;
```

Comment:
You can be cleaner here and below and `return result.value_or(default_setting);`